packages/core-components/src/components/LogViewer/RealLogViewer.tsx

```diff
@@ -47,11 +49,27 @@ export function RealLogViewer(props: RealLogViewerProps) {
   const selection = useLogViewerSelection(lines);
   const location = useLocation();
 
+  const ref$ = new BehaviorSubject<FixedSizeList | null>(
+    fixedSizeListRef.current,
+  );
+  const listRef = useObservable(ref$);
```

Comment:
You don't really need the subject or an observable for this right? looks like something you could implement with `useState` and/or `useRef`